OmniBLE/Bluetooth/PeripheralManager.swift

```diff
@@ -460,17 +506,34 @@ extension PeripheralManager: CBPeripheralDelegate {
 
 extension PeripheralManager: CBCentralManagerDelegate {
     func centralManagerDidUpdateState(_ central: CBCentralManager) {
+        self.log.debug("PeripheralManager - centralManagerDidUpdateState: %@", central)
         switch central.state {
         case .poweredOn:
```

Comment:
Can you add  some comments on you expect these `isSuspended` changes to affect the flow?